auth-web/src/components/auth/manage-business/AffiliatedEntityTable.vue

```diff
@@ -178,11 +224,43 @@ export default class AffiliatedEntityTable extends Mixins(DateMixin) {
     return this.headers?.filter(x => x.show)
   }
 
+  private get getStatuses (): string[] {
+    const items = []
+    for (const i in this.filteredItems) {
+      if (!items.includes(this.status(this.filteredItems[i]))) {
+        items.push(this.status(this.filteredItems[i]))
+      }
+    }
+    return items
+  }
+
+  private get getTypes (): string[] {
+    const items = []
+    for (const i in this.filteredItems) {
+      if (!items.includes(this.type(this.filteredItems[i]))) {
+        items.push(this.type(this.filteredItems[i]))
+      }
+    }
+    return items
+  }
+
   /** The set height when affiliation count exceeds 5 */
   private get getMaxHeight (): string {
     return this.entityCount > 5 ? '32rem' : null
   }
 
+  private get showClearFilters (): boolean {
+    for (const key in this.filterActive) {
```

Comment:
would `includes` work?
`this.filterActive.includes(key)`

Reply:
no because filterActive is an obj

Reply:
My gut feeling tells me there is a one liner to do this

Reply:
adding this instead: `return Object.values(this.filterActive).includes(true)`